studio/components/interfaces/App/FeaturePreview/APISidePanelPreview.tsx

```diff
@@ -0,0 +1,48 @@
+import { BASE_PATH } from 'lib/constants'
+
+const APISidePanelPreview = () => {
+  return (
+    <div className="space-y-2">
+      <p className="text-foreground-light text-sm">
+        Get building with your client application faster with your Project's API documentation now
```

Comment:
@saltcod wondering if you might have some thoughts about the copy writing here - feels like these are a bit wordy but i'm not sure how to concisely describe the changes haha

Reply:
Added a bunch of notes to notion! I think my biggest takeaway is we should cut a ton of this to the bone! API panel and this feature preview panel alike.

Reply:
Maybe an even more condense version of the top copy here and X all of the bottom stuff (or more likely, move the top stuff to the bottom) 

https://share.cleanshot.com/nsV61Qb1

(+ add a "Give Feedback" button to link to the Discussion we'll create, as mentioned)

Reply:
> Added a bunch of notes to notion! I think my biggest takeaway is we should cut a ton of this to the bone! API panel and this feature preview panel alike.

gotcha on this! will go through em and see what can be actioned on immediately :)

Reply:
> X all of the bottom stuff

i'm thinking about this though - should we? github doesnt have this section but i'm thinking it might be worth pointing out what are the UI changes once they've enabled so there won't be confusion as to things moving/changing

we could probably leave out _what_ the feature preview does though which is what most of the text is about, but i'm thinking we keep things like "GraphQL interface will shift to database page", API docs button shifts to bottom of left nav bar

Reply:
so maybe just like this
![image](https://github.com/supabase/supabase/assets/19742402/671a2b8b-b663-402d-8ccd-27d257bad9a5)